database/migrations/2021_11_21_215954_change_json_stats_size_for_equipment_mods.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeJsonStatsSizeForEquipmentMods extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('equipment_mods', function (Blueprint $table) {
+            $table->string('json_stats', 2500)->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('equipment_mods', function (Blueprint $table) {
+            $table->string('json_stats', 2500)->change();
```

Comment:
I think all of these should be 1000, right?

Reply:
Good catch thank you for fixing! Sorry I was afk